src/locale/lang/fr.js

```diff
@@ -0,0 +1,85 @@
+export default {
+  el: {
+    datepicker: {
+      now: 'Maintenant',
+      today: "Aujourd'hui",
```

Comment:
"Auj." (with the dot at the end) could be used. It's popular.
"Mnt" is one possible abbreviation for "Maintenant", but it sounds weird aside the "OK" button.

"Sélectionner" could be replaced by "Choisir".